Sources/PluginViewController/Configurations/SectionsConfiguration.swift

```diff
@@ -1,55 +1,135 @@
 //
 //  Copyright © 2017 Nikita Ermolenko. All rights reserved.
+//  Copyright © 2019 Rosberry. All rights reserved.
 //
 
 import Foundation
 
 final class SectionsConfiguration: NSObject, Configuration {
 
-    let sections: [Section]
+    let plugins: [Plugin]
+
+    var sections: [NavigationPlugin] = []
+    var tableViewItems: [PluginItem] = []
+
     weak var tableView: UITableView?
+    weak var delegate: ConfigurationDelegate?
 
-    init(sections: [Section]) {
-        self.sections = sections
+    init(plugins: [Plugin]) {
+        self.plugins = plugins
     }
 
     func configure() {
-        sections.flatMap { section in
-            return section.plugins
-        }.forEach { plugin in
-            self.tableView?.register(plugin.nib, forCellReuseIdentifier: plugin.cellIdentifier)
+        sections = self.sections(for: plugins)
+        tableViewItems = sections.map({ (section: NavigationPlugin) -> PluginItem in
```

Comment:
Is it possible to omit `(`?
```suggestion
        tableViewItems = sections.map { (section: NavigationPlugin) -> PluginItem in
```